pages/instructorAssessmentQuestionManualGrading/instructorAssessmentQuestionManualGrading.ejs

```diff
@@ -2,18 +2,272 @@
+                      filterControl: 'input',
+                      sortable: true,
+                      visible: <%= !assessment.group_work %>,
+                  },
+                  {
+                      field: 'requires_manual_grading',
+                      title: 'Grading status',
+                      filterControl: 'select',
+                      // filterDefault: 'true',
+                      searchFormatter: false,
+                      sortable: true,
+                      class: 'text-center',
+                      formatter: (value, row) => row.requires_manual_grading ? 'Requires grading' : 'Graded',
+                  },
+                  {
+                      field: 'assigned_grader',
+                      title: 'Assigned grader',
+                      filterControl: 'select',
+                      searchFormatter: false,
+                      formatter: (value, row) => row.assigned_grader ? row.assigned_grader_name : '&mdash;',
+                  },
+                  {
+                      field: 'points',
+                      title: 'Awarded points',
+                      class: 'text-center',
+                      filterControl: 'input',
+                      searchFormatter: false,
+                      sortable: true,
+                      formatter: pointsFormatter,
+                  },
+                  {
+                      field: 'score_perc',
+                      title: 'Percentage score',
+                      class: 'text-center align-middle text-nowrap',
+                      filterControl: 'input',
+                      searchFormatter: false,
+                      sortable: true,
+                      formatter: scorebarFormatter,
+                  },
+                  {
+                      field: 'last_grader',
+                      title: 'Graded by',
+                      filterControl: 'select',
+                      searchFormatter: false,
+                      formatter: (value, row) => row.last_grader ? row.last_grader_name : '&mdash;',
+                  },
+              ]],
+          });
       });
+
+      function ajaxSubmit(e) {
+          e.preventDefault();
+          const submitter = e.submitter || e.originalEvent?.submitter;
+          if (submitter) {
+              // Obtain information from submit button that was used
+              $(e.target).find('input[name=batch_action_data]')
+                  .val(JSON.stringify($(submitter).data('batch-action') || {}));
+          }
+          $.post($(e.target).attr('action'), $(e.target).serialize(), function(data) {
+              $('#grading-table').bootstrapTable('refresh');
+          }, 'json').fail(function(data) {
+              // TODO Better user notification of update failure
+          });
+      }
+
+      function gradingTagDropdown() {
+          const course_staff = JSON.parse('<%- JSON.stringify(course_staff) %>') || [];
+          const dropdown = $('<div class="dropdown ml-1">');
```

Comment:
I wanted to ensure the button and the dropdown are in the same div separate from other buttons to avoid issues later on if another button with a dropdown is included (e.g., to select columns). I figured out that it's possible to add a btn-group inside another btn-group to take care of this, should be ok now.